### DISCUSSION TITLE
Site Reporting to PDF

### DISCUSSION THREAD

**Discussion by Jarli01** [amidaware/tacticalrmm#1031] [Q&A] [answered]
This is a general ask of what kind of reporting is available from the system. I see that Grafana integration is available as a 3-party addon. What I am wondering is possible is to get a site's monthly stats to report to an PDF document of sorts, showing things like uptime stats, patches installed etc. 

In the demo I haven't been able to find anything quite like this so I'm not sure if there is another solution that is recommended or if this is potentially a WIP.

**Comment by dinger1986** [comment]
hello native reporting is currently WIP but looking good

**Comment by bbrendon** [comment, marked as answer]
Hi @Jarli01  
The Grafana thing you found is the best turn-key thing currently available.  Someone is working on an official reporting tool but no ETA.

The data is in SQL so the possibilities are endless to roll your own.
_BB

**Comment by Jarli01** [comment]
Thanks for the quick replies. Out of curiosity where can I see the WIP?

**Reply by bbrendon** [reply]
The developer hasn't shared it yet.

**Reply by Jarli01** [reply]
Shoot, well thank you for the information provided.

**Comment by ebdavison** [comment]
I would be interested as well in the WIP code and am willing to contribute to the effort if that is possible.  

I have seen the "Collect System Report and Email" but it requires the SMTP server and password and such and is not as user friendly as I would expect.  It would be good if it could use the SMTP setting in the global config and not have to specify all of that stuff.  Possibly attach the report to the agent itself as a Note even.

**Reply by silversword411** [reply]
> I would be interested as well in the WIP code and am willing to contribute to the effort if that is possible.

What is available thus far https://github.com/amidaware/tacticalrmm/pull/1086
